docs/multiple-ako.md

```diff
@@ -0,0 +1,69 @@
+# Multiple AKO instances in a cluster
+
+This feature allows to run multiple instances of AKO per cluster.
+
+## Overview
+
+Prior to AKO 1.7.1, single instance of AKO was responsible to process kubernetes/openshift objects updates and creates corresponding AVI objects at controller.
```

Comment:
suggesting.
~and creates corresponding AVI objects at controller.~ and synced corresponding objects in the Avi Controller.